index.html

```diff
@@ -37,6 +37,123 @@
         <span class="navbar-toggler-icon"></span>
       </button>
 
+
```

Comment:
You should start writing after 114 line of code

Reply:
@vikhyatsingh123 sorry for the inconveniences.I would make PR again